GDJS/Runtime/debugger-client/hot-reloader.ts

```diff
@@ -174,6 +176,17 @@ namespace gdjs {
         const newRuntimeGameOptions: RuntimeGameOptions =
           gdjs.runtimeGameOptions;
 
+        this._runtimeGame._displayMessageInPreview =
```

Comment:
I'm a bit uneasy at this patching of a private variable + patching of _options.
Could we rather call a method, like `runtimeGame.displayInAppTutorialMessage(...)`?